Firestore/Swift/Tests/Integration/CodableIntegrationTests.swift

```diff
@@ -67,7 +67,8 @@ class CodableIntegrationTests: FSTIntegrationTestCase {
         return nil
       }) { object, error in
         completion?(error)
-    } }
+      }
```

Comment:
You likely have the wrong version of swiftformat installed if you're generating diffs like these.

https://github.com/firebase/firebase-ios-sdk/blob/master/README.md#code-formatting

Reply:
That's weird. I enter the cmd you linked and my script/style.sh still doesn't revert the change. anyway, I manually revert this file.